insight.py

```diff
@@ -95,13 +95,14 @@ def collector(self, args):
         if args.pid:
             logging.debug(
                 "Collecting process infor only for PID %s" % args.pid)
-            collector_exec = [collector_exec, '-pid', '%s' % args.pid]
+            collector_exec = [collector_exec, '-proc', '-pid', '%s' % args.pid]
```

Comment:
How is `-proc`'s value passed in?

Reply:
`-proc` is a Boolean flag, and no value is needed for it. (default is `false`, set `-proc` makes it `true`, and it's also possible to pass a specific value to it, but that's not necessary)

Reply:
Ok, I understand.